packages/slice-machine/components/AppLayout/Navigation/index.tsx

```diff
@@ -32,6 +33,15 @@ const links: LinkProps[] = [
     },
     Icon: MdHorizontalSplit,
   },
+  {
+    title: "Changes",
+    href: "/changes",
+    match(pathname: string) {
+      return pathname.indexOf("/changes") === 0;
+    },
+    Icon: MdLoop,
+    numberOfChanges: 5,
```

Comment:
Don't know if that's the best way of doing it because `numberOfChanges` is a prop very specific to just one nav item, but has nothing to do with the others. I would have maybe made it more generic like `RightElement: <ChangesIndicator numberOfChanges={link.numberOfChanges} />`

then in the Item component it could've been: 
```
<Box as="span" sx={{ ml: 2, fontWeight: 400 }}>
                {link.title}
              </Box>
              {RightElement && <RightElement/>
```

That way if the design require new tabs with different icons/elements we can just use that instead of having to cerate a new prop for each

Reply:
what do you think?

Reply:
this is a good point, i'll make it more generic!